go/vt/vtctl/vtctl.go

```diff
@@ -2486,7 +2486,7 @@ func commandVRWorkflow(ctx context.Context, wr *wrangler.Wrangler, subFlags *fla
 		vrwp.Cells = *cells
 		vrwp.TabletTypes = *tabletTypes
 		if vrwp.TabletTypes == "" {
-			vrwp.TabletTypes = "primary,replica,rdonly"
+			vrwp.TabletTypes = "in_order:REPLICA,PRIMARY"
```

Comment:
Was `rdonly` removed  on purpose?

Reply:
Yeah, according to our docs were use the default of the tablet flag if you don't specify and that's what we're actually now doing here. After this PR, the only place RDONLY is potentially used by default is for VDiff.

Let me know if not using the default was intentional before and perhaps the docs were incorrect.

Reply:
Related to this, I'm not sure why are have this `if vrwp.TabletTypes == "" {` code block at all since we are specifying a default value here: https://github.com/vitessio/vitess/pull/10040/files#diff-ad37010accc5a29c6751d18124328bb882457e8b2527db688b854c81fe23861cR2333

Maybe that wasn't always working as I'd expect.

Reply:
@rohit-nayak-ps turns out that you were right! I shouldn't have removed that. Re-adding it fix this: https://github.com/vitessio/vitess/issues/10421

:facepalm:

Reply:
Well, after further testing... this didn't actually change any behavior. We were never switching `RDONLY` traffic by default -- as the `tablet_types` flag variable was never an empty string as it has a default value (which is what we changed) -- and we weren't testing for this either. Will correct both in https://github.com/vitessio/vitess/issues/10421.